### ISSUE TITLE
/l and /ls do not respect Accept: application/json

### ISSUE THREAD

**Issue by HalosGhost** [ptpb/pb#138]
```console
$ curl -H 'Accept: application/json' https://ptpb.pw/ls
- paraiso-light
- manni
- monokai
- igor
- emacs
- bw
- fruity
- murphy
- autumn
- vs
- vim
- tango
- native
- perldoc
- default
- pastie
- borland
- xcode
- trac
- rrt
- colorful
- friendly
- paraiso-dark
```

``/l`` has the same issue. Both of these are using ``complex_response()``, why are these routes not respecting ``Accept: application/json``?

**Comment by buhman** [comment]
Heh. They are, but because this is a GET, you get the cached yaml response instead.

**Comment by buhman** [comment]
Turns out varnish already has behavior for dealing with this by default; fixed in 0e2632575bf788681f6e934fa9b666ae57042fa5

**Comment by HalosGhost** [comment]
\o/
